### ISSUE TITLE
data issues page update

### ISSUE THREAD

**Issue by zcrysler** [MotusDev/Motus-TO-DO#161]
Can we have the motusTagID listed next to all tag issues? It's hard to follow what changes your making, particularly to "tag detections outside of deployments", when the tag isn't identified in a long list of issues.

Same for receivers, there should be some sort of identifier with all issues so that it's easy to follow, eg. GPS fixes far from deployment location - would be helpful know what receivers are popping up here without having to open a new link for each error

**Comment by cjardine-bsc** [comment]
Jeremy this is already in the error table in the database under ID. 

This ID should be included in the tables for all error codes (not just tags but also sensors). 

Thanks!

**Comment by Hussell** [comment]
I've changed sandbox to start feeding the various ID fields in the database to the error messages, and edited most of the messages on sandbox. Have a look and let me know if the changes are what you were looking for.

Putting these edits in the live database will have to wait until after the next deployment to live, otherwise the live site will break.
